### PR TITLE
[RFR][BC-BREAK] Remove referencedView created by code for ReferenceField

### PR DESCRIPTION
**Pull request by jeromemacias**
* [x] Remove creation of `ListView` in `ReferenceField`
* [x] Fix sort attributes for `ma-datagrid` directive
* [x] Fix tests
* [x] Add note about upgrade and BC-Break

### CODE REVIEW

**Comment by jeromemacias** [comment]
Tests are OK (see 3rd commit).

File: UPGRADE-0.8.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Upgrade to 0.8
+
+## View and Entity URL customization
+
+Arguments for the anonymous function of the entity url customisation was changed.
```
Comment (fzaninotto) [inline UPGRADE-0.8.md]: s/was/were

**Comment by fzaninotto** [comment]
I'm OK for merge, but I'd like to know which PR should be merged first, yours or @manuquentin's?

**Comment by jeromemacias** [comment]
As we discuss with @manuquentin, mine should be merged first, I let him confirm.

**Pull request merged**